### DISCUSSION TITLE
工作区布局截图分享

### DISCUSSION THREAD

**Discussion by Quorafind** [obsidianzh/forum#33] [General综合]
每个人都有各自喜欢的工作区布局，有些工作布局时为了自己在工作的时候可以快速地刺激自己大量书写，有一些则是能让自己快速学习相关的内容，你有什么好的工作区布局可以分享吗？

**Comment by makerjackie** [comment]
我通常左边 文件浏览器+大纲，右边 标签 + backlink

大部分情况下，通过快捷键（可以自行设置）ctrl + L（left）ctrl + R （right）隐藏左右侧边栏

**Reply by Quorafind** [reply]
类似，不过我右边多了个日历
